README.md

```diff
@@ -2,26 +2,35 @@
 This project creates a `wicked2nm` binary which is able to parse wicked xml
 configs and send them to a NetworkManager dbus service.
 ## Installation
-The main way to use the migration is through the container image available at
-`registry.opensuse.org/home/jcronenberg/migrate-wicked/containers/opensuse/wicked2nm:latest`.  
-For openSUSE a package that includes all the latest changes is available at
+`wicked2nm` is available in openSUSE tumbleweed `zypper in wicked2nm`.
+
+There is also a package that includes all the latest changes available at
 https://build.opensuse.org/package/show/home:jcronenberg:migrate-wicked/wicked2nm.
 ## Usage
-The main recommended way to use `wicked2nm` is with a container.
+### On system
+If both `wicked` and `NetworkManager` are available `wicked2nm` can be run on the system directly.
+```bash
+# NetworkManager-config-server is required as otherwise NM will add connections, resulting in duplicates.
+# NetworkManager-config-server can be removed after the migration is done.
+zypper in wicked2nm NetworkManager NetworkManager-config-server
+
+# You can test beforehand whether there are errors or warnings.
+wicked show-config | wicked2nm migrate --dry-run -
+
+# Be careful here as wicked will shut down the interfaces and they will only come up again once the migration is done.
+# This oneliner shuts down wicked, starts NM and runs the migration, if anything went wrong it starts wicked again.
+systemctl disable --now wicked \
```

Comment:
I think a short version as before with 
```
wicked show-config > wicked.xml
wicked2nm migrate wicked.xml
```
is more readable. And maybe should be in this doc as well, plus the one-liner which does everything.
Just to show what is actually needed by `wicked2nm`, wdyt?

Reply:
Okay, I updated it. Do you think it should also be noted that this supposed to be run as root?